{
  "uid" : "73f31f69da71acab",
  "name" : "doesn't accept ether transfers",
  "fullName" : "Contract: GovernorTimelockControl.doesn't accept ether transfers",
  "historyId" : "67e7d81e4985e79ee8275d21814a81f3",
  "time" : {
    "start" : 1635782688377,
    "stop" : 1635782689347,
    "duration" : 970
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0689340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0689340>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/governance/extensions/GovernorTimelockControl.test.js:30:33)\n                    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: GovernorTimelockControl"
  }, {
    "name" : "testClass",
    "value" : "Contract: GovernorTimelockControl"
  }, {
    "name" : "testMethod",
    "value" : "doesn't accept ether transfers"
  }, {
    "name" : "package",
    "value" : "Contract: GovernorTimelockControl"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ {
      "uid" : "f25916751e7da346",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0637f40>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782587172,
        "stop" : 1635782587732,
        "duration" : 560
      }
    }, {
      "uid" : "dc80806c0c95edcb",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0fe62e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782585521,
        "stop" : 1635782586087,
        "duration" : 566
      }
    }, {
      "uid" : "bf28bfb6cd84493c",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a04356d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782585148,
        "stop" : 1635782585719,
        "duration" : 571
      }
    }, {
      "uid" : "cd6242ef862b5138",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a2856ee0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782539364,
        "stop" : 1635782539928,
        "duration" : 564
      }
    }, {
      "uid" : "6b251e1149d8bf51",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0452250>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782528513,
        "stop" : 1635782529483,
        "duration" : 970
      }
    }, {
      "uid" : "8d25ad2aeafe6c0f",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0437400>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782506771,
        "stop" : 1635782507332,
        "duration" : 561
      }
    }, {
      "uid" : "f5cbc58392aa3dd3",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f238be9eac0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782494794,
        "stop" : 1635782495359,
        "duration" : 565
      }
    }, {
      "uid" : "984052b570797747",
      "status" : "broken",
      "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a17aad60>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
      "time" : {
        "start" : 1635782492210,
        "stop" : 1635782492772,
        "duration" : 562
      }
    } ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "52c6d54880e36754",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/8//#testresult/52c6d54880e36754",
        "status" : "broken",
        "statusDetails" : "Returned error: Transaction simulation failed: Error processing Instruction 0: invalid account data for instruction",
        "time" : {
          "start" : 1635589822496,
          "stop" : 1635589838206,
          "duration" : 15710
        }
      }, {
        "uid" : "e5cb0172007673ef",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/7//#testresult/e5cb0172007673ef",
        "status" : "broken",
        "statusDetails" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a28660a0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
        "time" : {
          "start" : 1635554964921,
          "stop" : 1635554965942,
          "duration" : 1021
        }
      }, {
        "uid" : "ec5d3bb407a1eeb5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/6//#testresult/ec5d3bb407a1eeb5",
        "status" : "broken",
        "statusDetails" : "HH110: Invalid JSON-RPC response received: <html>\n                <head><title>502 Bad Gateway</title></head>\n                <body>\n                <center><h1>502 Bad Gateway</h1></center>\n                <hr><center>nginx/1.18.0 (Ubuntu)</center>\n                </body>\n                </html>\n                ",
        "time" : {
          "start" : 1635551252618,
          "stop" : 1635551252725,
          "duration" : 107
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "73f31f69da71acab.json",
  "parameterValues" : [ ]
}